geth/jail/execcution_policy.go

```diff
@@ -0,0 +1,196 @@
+package jail
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+	"time"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/common"
+)
+
+// ExecutionPolicy provides a central container for the executions of RPCCall requests for both
+// remote/upstream processing and internal node processing.
+type ExecutionPolicy struct{}
```

Comment:
In all `ExecutionPolicy` methods, there is `manager common.NodeManager`. Maybe we could put it into the struct fields? 

```go
type ExecutionPolicy struct {
  manager common.NodeManager
  account common.AccountManager
}
```

and provide `func NewExecutionPolicy()` constructor? 

Function signatures should be much shorter then.